Tasks/Common/utility-common/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "utility-common",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Should be 1.0.2

Reply:
I think you are confusing this package which lives in , utility-common, with bumped versions in nuget-task-common.  utility-common moved from 1.0.0 to 1.0.1 and nuget-task-common moved from 1.0.1 to 1.0.2

Reply:
Ah yes you're right. And now that you mention it, the telemetry module needs to be compiled with the task, this is done through the task's make.json file.

Reply:
Moot point now.  The common module dependency no longer exists.